### ISSUE TITLE
Are the tests working?

### ISSUE THREAD

**Issue by pecalleja** [xlwings/jsondiff#58]
I cloned the project and executed the tests but it's not working. do you have any advice?
```
$ pytest -vv
```
output 
```
self = <tests.test_jsondiff.JsonDiffTests testMethod=test_a>

    def test_a(self):

        self.assertEqual({}, diff(1, 1))
        self.assertEqual({}, diff(True, True))
        self.assertEqual({}, diff('abc', 'abc'))
        self.assertEqual({}, diff([1, 2], [1, 2]))
        self.assertEqual({}, diff((1, 2), (1, 2)))
        self.assertEqual({}, diff({1, 2}, {1, 2}))
        self.assertEqual({}, diff({'a': 1, 'b': 2}, {'a': 1, 'b': 2}))
        self.assertEqual({}, diff([], []))
        self.assertEqual({}, diff(None, None))
        self.assertEqual({}, diff({}, {}))
        self.assertEqual({}, diff(set(), set()))

        self.assertEqual(2, diff(1, 2))
        self.assertEqual(False, diff(True, False))
        self.assertEqual('def', diff('abc', 'def'))
        self.assertEqual([3, 4], diff([1, 2], [3, 4]))
        self.assertEqual((3, 4), diff((1, 2), (3, 4)))
        self.assertEqual({3, 4}, diff({1, 2}, {3, 4}))
>       self.assertEqual({replace: {'c': 3, 'd': 4}}, diff({'a': 1, 'b': 2}, {'c': 3, 'd': 4}))
E       TypeError: unhashable type: 'Symbol'

tests/test_jsondiff.py:45: TypeError
```

**Comment by ramwin** [comment]
I met the same problme.

**Comment by corytodd** [comment]
This was broken by #55 and fixed by #61.
